shorten-name/name.go

```diff
@@ -2,8 +2,17 @@ package main
 
 import (
 	"fmt"
+	"strings"
 )
 
 func main() {
-	fmt.Println("Array!")
+	fullNameArray := []string{"Nibras Ahmed", "Ahmed Saleem", "Ahsan Yousaf", "Muhammad Husnain"}
+	shortNameArray := []string{}
+	for _, name := range fullNameArray {
+		parts := strings.Split(name, " ")
```

Comment:
Donot use Split function or any build in packages like "strings"

Reply:
Alright, is it good now?